labs/00-labs-01-getting_started/2.md

```diff
@@ -1,40 +1,36 @@
 **Figure: Example of local user directory**
 
 ![](assets/images/working_directory.png)
 
-The top level of your local repo must contain a `labs/` folder that represents the same-named folder in the DevNet GitLab repo. Each learning lab that you create resides in its own folder under the `labs/` directory of your local repository.  The following image shows many learning labs as subdirectories of the `labs/` folder; for example, a lab named `01-intro-01-intro-to-devnet` appears near the top of the illustration. The address bar in the illustration shows that this folder and its siblings are children of the `labs/` folder. You can also see the use of a naming convention that DevNet staff uses: the `01-intro-01` stands for "module 1 intro lab 1" followed by a brief lab title.
+The top level of the repository must contain a `labs/` folder that represents the same-named folder in the DevNet GitLab repo. Each learning lab that you create resides in its own folder under the `labs/` directory of your local repository.  The following image shows many learning labs as subdirectories of the `labs/` folder; for example, a lab named `01-intro-01-intro-to-devnet` appears near the top of the illustration. The address bar in the illustration shows that this folder and its siblings are children of the `labs/` folder. You can also see the use of a naming convention that DevNet staff uses: the `01-intro-01` stands for "module 1 intro lab 1" followed by a brief lab title.
 
 **Figure: Individual labs as children of the labs/ folder in  local repo**
-
 ![](assets/images/labs_dir.png)
 
 > **Note:** The **labs/** directory must be the parent directory of all labs in your repository&mdash;the publishing system searches for the **labs/** directory when attempting to publish labs to the production environment.
 
-An individual learning lab resides in a directory that is an immediate child of the labs/ directory; in this example,  `01-intro-01-intro-to-devnet` is this directory, which we will call the "_your-learning-lab_ directory." Inside the _your-learning-lab_ directory, create a directory named __assets__; inside the __assets__ directory, create a directory named __images__.  As you create images for your learning lab, place the image files in the labs/_your-learning-lab_/assets/images directory. Note that images should be PNG format and 300 KB or less in size.
+An individual learning lab resides in a directory that is an immediate child of the `labs/` directory. In this example,  `01-intro-01-intro-to-devnet` is this directory, which we will call the "_your-learning-lab_ directory." Inside the _your-learning-lab_ directory, create a directory named __assets__; inside the __assets__ directory, create a directory named __images__.  As you create images for your learning lab, place the image files in the labs/_your-learning-lab_/assets/images directory. Note that images should be PNG format and 300 KB or less in size.
```

Comment:
Eh, we should probably scrub for capitalizing Learning Labs as if it's a product.